mainapp/templates/mainapp/fullscreen_elements.html

```diff
@@ -0,0 +1,14 @@
+    <text transform="matrix(1 0 0 1 77 37)" fill="#575756" font-family="Arial" font-size="12">N</text>
+</g>
```

Comment:
What is the source of this svg picture?

Reply:
I don't see any indication of a public domain grant for that image. Better use a source that is explicitly permitting unrestricted use such as [this CC0 image](https://commons.wikimedia.org/wiki/File:Gray_compass_rose.svg) or any other [CC0 windrose SVG from Wikimedia Commons](https://commons.wikimedia.org/w/index.php?search=compass+rose&title=Special%3AMediaSearch&type=image&haslicense=unrestricted&filemime=svg).